FirebasePerformance/Sources/Instrumentation/FPRNetworkTrace.m

```diff
@@ -141,8 +141,8 @@ - (void)updateTraceWithCurrentSession {
   if (self.traceStarted && !self.traceCompleted) {
     dispatch_async(self.sessionIdSerialQueue, ^{
       FPRSessionManager *sessionManager = [FPRSessionManager sharedInstance];
-      FPRSessionDetails *sessionDetails = sessionManager.sessionDetails;
-      if (sessionDetails) {
+      FPRSessionDetails *sessionDetails = [sessionManager.sessionDetails copy];
+      if (sessionDetails && [sessionDetails isKindOfClass:[FPRSessionDetails class]]) {
```

Comment:
Isn't the `isKindOfClass` check redundant after the copy?

Reply:
It is redundant. Let me remove it.

Reply:
Done.